db/schema.rb

```diff
@@ -63,6 +64,12 @@
     t.index ["updated_at"], name: "index_orm_resources_on_updated_at"
   end
 
+  create_table "processed_events", force: :cascade do |t|
```

Comment:
Where is this coming from?  I don't see anything in this PR — is this a change from another issue, or a side effect I'm not seeing?

Reply:
This is related to https://github.com/pulibrary/figgy/pull/2039/files#diff-81be10e318670d2eefc0f09ce40e1179.  Normally `ScannedResources` stored the `token` attribute for `AuthToken` objects into the `auth_token` attribute.  However, when `ScannedResources` are taken down, the `auth_token` attribute is removed.  The `AuthToken` object remains persisted.  Once the `ScannedResource` is marked as complete at a later point in time, this is used to retrieve the existing `AuthToken` and re-link it to this `ScannedResource`.

One error I've just found here is that there may be a case where a taken down `ScannedResource` may then be deleted; under those circumstances, I don't believe that the `AuthToken` would be deleted.

Reply:
Rather than de-link, would it be easier to just make the auth token not work on incomplete resources?

Reply:
So the auth token would override resource visibility, but not state-based limits to visibility?  That seems reasonable to me — and would allow us to simply leave the auth token in place when a resource was taken down, and not need to delete/re-add later.